realhf/impl/model/utils/grouped_gemm.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
+
+try:
+    import grouped_gemm
+except ImportError:
+    grouped_gemm = None
+
+
+def grouped_gemm_is_available():
```

Comment:
Move it into constants?

Reply:
I think they should be moved into experts.py since they are not used anywhere else.